CHANGES.rst

```diff
@@ -172,6 +172,12 @@ Bug Fixes
 
 - ``astropy.utils``
 
+  - Fix to allow the C-based _fast_iterparse() VOTable XML parser to
+    relloc() its buffers instead of overflowing them.
+    (Bug occuring when Gzip-compressed VOTable XML streams are passed to
```

Comment:
Still think that there is no need for the parenthesis. Hardly any of the other entries in the changelog have such detailed description, and since the PR/issue number is listed one can always easily find further detail if needed following those links.